src/System.Text.Primitives/System/Text/Encoding/SymbolTable.cs

```diff
@@ -107,6 +107,77 @@ public bool TryParse(ReadOnlySpan<byte> source, out Symbol symbol, out int bytes
 
         public abstract bool TryParse(ReadOnlySpan<byte> source, Span<byte> utf8, out int bytesConsumed, out int bytesWritten);
 
+        #region Public UTF-16 to UTF-8 helpers
+
+        public bool TryEncode(ReadOnlySpan<char> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten)
```

Comment:
That would conflict with the UTF-8 version... and this is specifically for UTF-16 strings.